libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailPasswordFragment.java

```diff
@@ -127,6 +128,14 @@ public void onPause() {
         mServiceEventConnection.disconnect(getContext(), this);
     }
 
+    private void updatePrimaryButtonEnabledStatus() {
+        View view = getView();
+        if (view != null) {
```

Comment:
Why exactly do we need to check this here?

Reply:
I've added this check because Android Studio complains that the view might be null with a warning. I'm fine with removing it if you think it doesn't make sense 👍

Reply:
I understand how the check is necessary for `LoginEmailFragment` since we're using the `view` to find a button:

```java
private void updateContinueButtonEnabledStatus() {
    View view = getView();
    if (view != null) {
        Button continueButton = (Button) view.findViewById(R.id.login_continue_button); // <-- We use the view here
        String currentEmail = mEmailInput.getEditText().getText().toString();
        continueButton.setEnabled(!currentEmail.trim().isEmpty());
    }
}
```

But for `LoginEmailPasswordFragment` or `LoginUsernamePasswordFragment` it looks like we are not using the `view` to do anything directly:

```java
private void updatePrimaryButtonEnabledStatus() {
    View view = getView();
    if (view != null) {
        // We don't use the view anywhere?
        String currentPassword = mPasswordInput.getEditText().getText().toString();
        getPrimaryButton().setEnabled(!currentPassword.trim().isEmpty());
    }
}
```

Are we keeping the check there just to make sure the Fragment's main view was already inflated or am I missing something?

Reply:
ah you're right, I think I used to use the view and removed it later. I'll fix this!